src/test/java/com/bandwidth/sdk/messaging/MessagingCallbackHelperTest.java

```diff
@@ -0,0 +1,45 @@
+package com.bandwidth.sdk.messaging;
+
+import com.bandwidth.sdk.messaging.models.ImmutableMessage;
+import com.bandwidth.sdk.messaging.models.ImmutableMessageEvent;
+import com.bandwidth.sdk.messaging.models.Message;
+import com.bandwidth.sdk.messaging.models.MessageEvent;
+import com.bandwidth.sdk.messaging.serde.MessageSerde;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class MessagingCallbackHelperTest {
+
+    private final Message message = ImmutableMessage.builder()
+            .addTo("1")
+            .addTo("2")
+            .applicationId("1")
+            .id("1")
+            .owner("3")
+            .segmentCount(1)
+            .text("1")
+            .time("1")
+            .from("1")
+            .direction("in")
+            .build();
+
+    private final MessageEvent messageEvent = ImmutableMessageEvent.builder()
+            .message(message)
+            .description("message-delivered")
+            .time("1")
+            .type("Message delivered to carrier")
+            .build();
+
+    private final MessagingCallbackHelper callbackHelper = new MessagingCallbackHelper();
+    private final MessageSerde serde = new MessageSerde();
+
+
+    @Test
+    public void testParseCallback() throws IOException {
+        MessageEvent returnedEvent = callbackHelper.parseCallback(serde.serialize(messageEvent));
+        Assert.assertEquals(returnedEvent,messageEvent);
```

Comment:
What do you think about using AssertJ for our assertions? This is what we use is most other projects, and they tend to read better / provide a bit more features
http://joel-costigliola.github.io/assertj/